definition/data/sheets/CaseField.json

```diff
@@ -90,5 +90,13 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "directionOrder", "Label": "Direction order", "FieldType": "DirectionOrder", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "channel", "Label": "Channel", "FieldType": "FixedList", "FieldTypeParameter": "ChannelType", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "intentionToProceedDeadline", "Label": "Intention To Proceed Deadline", "FieldType": "Date", "FieldTypeParameter": "ChannelType", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "state", "Label": "state", "FieldType": "Text", "SecurityClassification": "Public"}
+  {"CaseTypeID": "MoneyClaimCase", "ID": "state", "Label": "state", "FieldType": "Text", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "issueLetterContact", "Label": "Who is the letter to?", "FieldType": "FixedRadioList", "FieldTypeParameter": "ChangeContactDetailsParty", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "letterContentHeader", "Label": "## Enter the text of the letter", "FieldType": "Label", "SecurityClassification": "Public"},
```

Comment:
Should this start with ## ? This seems like sample text that should be replaced by the user rather than a label.

Reply:
this is based non design approved by Marc and they are label